### DISCUSSION TITLE
Where did the custom shortcuts go?

### DISCUSSION THREAD

**Discussion by Test-Account666** [Floorp-Projects/Floorp#1059] [Q&A]
Hey!

I realized I wasn't using the latest version of floorp and updated.

Well... Now there's an entire settings category missing:
The custom shortcuts.

Did they move somewhere else?
My configured shortcuts still appear in `about:config`, but they're not working

**Comment by CutterKnife** [comment]
Strangely, it looks fine to me.

**Reply by regularentropy** [reply]
Can confirm. @TheBlackEntity Please try new profile

**Reply by Test-Account666** [reply]
A new profile also does not change this :/
![Missing Entry](https://github.com/Floorp-Projects/Floorp/assets/36412486/3deb8593-14a3-47e9-8f4e-cec31f2fd6a9)

**Comment by regularentropy** [comment]
> A new profile also does not change this :/ ![Missing Entry](https://private-user-images.githubusercontent.com/36412486/318285791-3deb8593-14a3-47e9-8f4e-cec31f2fd6a9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE5MDM2OTMsIm5iZiI6MTcxMTkwMzM5MywicGF0aCI6Ii8zNjQxMjQ4Ni8zMTgyODU3OTEtM2RlYjg1OTMtMTRhMy00N2U5LThmNGUtY2VjMzFmMmZkNmE5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzMxVDE2NDMxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE5MTVkYmQ5ZTdmY2I5OTg2NmU1ZmFkMWNjOTZhNTMzNDExOTliMWVmNzU2OTA4MmEzMDVjZDQzNmZjZTM2ZDQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.O8abTMi6H2QwBV84NBsVbzrf-WOPfE8AQF4PT8P6uyk)

Maybe try clean install then.

**Reply by Test-Account666** [reply]
A fresh reinstall also didn't help :/

I'm on Linux, if that matters

**Reply by CutterKnife** [reply]
It could be a problem that depends on a specific package. Please try changing it.
